snapd.go

```diff
@@ -422,7 +457,6 @@ func action(ctx *cli.Context) {
 	if autodiscoverPath != "" {
 		log.Info("auto discover path is enabled")
 		paths := filepath.SplitList(autodiscoverPath)
-		c.SetAutodiscoverPaths(paths)
```

Comment:
If it was a recent addition that had a clear plan to be used in the near future I'd be inclined to agree but this has been in and unused for 9 months. I don't think the fact that it would need to be added back in at a later date is a compelling argument for leaving it in. The removed code contains no logic that would make implementing those future changes any easier.